nimbus/Cargo.lock

```diff
@@ -1235,6 +1242,19 @@ dependencies = [
  "proc-macro2",
 ]
 
+[[package]]
```

Comment:
I assume this is fine since we don't need high-quality random numbers, but I'd be nice if we could make `rc_crypto` available to other rust projects :/

Reply:
I think this dep is just for `temp_dir` - that's the only new package I added.